miio/miioprotocol.py

```diff
@@ -90,18 +90,18 @@ def send_handshake(self, *, retry_count=3) -> Message:
         return m
 
     @staticmethod
-    def discover(addr: str = None, timeout: int = 5) -> Any:
+    def discover(addr: str = None, is_broadcast: bool = True, timeout: int = 5) -> Any:
```

Comment:
Could you also add the new param (and the missing timeout) to the docstring?

Reply:
Done. Hopefully this makes sense